tests/data/validation/enum/enum.go

```diff
@@ -324,6 +404,16 @@ func (j *EnumMyNumberUntypedEnum) UnmarshalJSON(b []byte) error {
 	return nil
 }
 
+// Verify checks all fields on the struct match the schema.
+func (plain EnumMyNumberUntypedEnum) Verify() error {
+	for _, expected := range enumValues_EnumMyNumberUntypedEnum {
+		if reflect.DeepEqual(v, expected) {
```

Comment:
where does this `v` come from? I don't see it declared anywhere, are you sure it's valid?